build.gradle

```diff
@@ -11,7 +12,7 @@ allprojects {
     apply plugin: 'checkstyle'
     apply plugin: 'com.diffplug.spotless'
 
-    group = 'com.amazon'
+    group = 'org.opensearch.dataprepper'
```

Comment:
Don't we need to update all sub-projects to use 'org.opensearch.dataprepper'?

Reply:
I'm not sure exactly what you mean.

By _convention_ the Maven groupId is the beginning of Java package names. Data Prepper 2.0 is scheduled to update all Java package names to `org.opensearch.dataprepper` as part of #344.

Because this project has not yet been deployed, the Maven groupId is currently not used. So this is not a breaking change.